test/conv_common.hpp

```diff
@@ -2631,3 +2631,27 @@ struct conv_bias_driver : test_driver
         verify(verify_backwards_bias<T>{output, bias});
     }
 };
+
+std::pair<miopen::Allocator::ManageDataPtr, size_t>
+AllocateConvTransposeWorkspace(miopen::Handle& handle,
+                               const miopen::TensorDescriptor& x,
+                               const miopen::TensorDescriptor& w,
+                               const miopen::TensorDescriptor& y)
+{
+
+    constexpr size_t alignment = 256u;
+
+    auto align = [&](size_t sz) {
+        auto ret = (sz + alignment - 1) & ~(alignment - 1);
+        assert(ret >= sz);
+        return ret;
+    };
```

Comment:
[Minor]
```suggestion
    constexpr size_t alignment = 256u;

    auto align = [](size_t sz) {
        static_assert(alignment > 0);
        constexpr size_t alignment_mask = alignment - 1;
        static_assert(alignment & alignment_mask  == 0);
        return (sz + alignment_mask ) & ~(alignment_mask );
    };
```
Runtime assertion is not required if `alignment` is not zero (which does not make sense in any way).
But `alignment` must be a power of 2, so few static assertion added.